sources/packages/api/src/services/education-program-offering/education-program-offering.service.ts

```diff
@@ -342,6 +349,46 @@ export class EducationProgramOfferingService extends RecordDataModelService<Educ
    * @returns offering object.
    */
   async getOfferingById(offeringId: number): Promise<EducationProgramOffering> {
-    return this.repo.findOne(offeringId);
+    return this.repo
+      .createQueryBuilder("offering")
+      .select([
+        "offering.id",
+        "offering.name",
+        "offering.studyStartDate",
+        "offering.studyEndDate",
+        "offering.actualTuitionCosts",
+        "offering.programRelatedCosts",
+        "offering.mandatoryFees",
+        "offering.exceptionalExpenses",
+        "offering.tuitionRemittanceRequestedAmount",
+        "offering.offeringDelivered",
+        "offering.lacksStudyDates",
```

Comment:
Are we still using this column? If am not wrong this is from the time that offering start/end dates were optional.